packaging_automation/tests/test_citus_package.py

```diff
@@ -71,7 +72,7 @@ def test_build_packages():
 
 def test_upload_to_package_cloud():
     output = upload_files_in_directory_to_package_cloud(BASE_OUTPUT_FOLDER, PLATFORM, PACKAGE_CLOUD_API_TOKEN, "sample",
-                                                        PACKAGING_EXEC_FOLDER)
+                                                        PACKAGING_EXEC_FOLDER, MAIN_BRANCH)
```

Comment:
maybe we can make `MAIN_BRANCH` a local variable of `test_upload_to_package_cloud` since it looks like we only use it here

Reply:
already fixed